fic/resource_eri_firewall_component.go

```diff
@@ -250,7 +250,10 @@ func resourceEriFirewallComponentV1Activate(d *schema.ResourceData, meta interfa
 	log.Printf("[DEBUG] RoutingGroupSettings  are set as: %#v", routingGroupSettings)
 
 	if len(rules) > 0 || len(customApplications) > 0 || len(applicationSets) > 0 || len(routingGroupSettings) > 0 {
-		updateFirewall(d, meta)
+		err := updateFirewall(d, meta)
+		if err != nil {
+			return fmt.Errorf("Error updating firewall component: %s", err)
+		}
```

Comment:
```suggestion
		if err := updateFirewall(d, meta); err != nil {
			return fmt.Errorf("Error updating firewall component: %s", err)
		}
```
ワンラインで書いていただいてもよろしいでしょうか？

Reply:
ご指摘ありがとうございます。updateFirewall関数を呼んでいるもう一箇所のほうも同じ修正をしておきます。